stories/form/TextField.stories.js

```diff
@@ -1,102 +1,85 @@
 import React from 'react';
```

Comment:
Some of these stories don't have `id` props passed in - I thought there was a bug with the implementation when Voiceover wasn't working, but then I realized it as just the component instances that were the problem.